<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>700</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Attack">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제우스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>포세이돈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아테나</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아프로디테</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아르테미스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아레스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디오니소스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데메테르</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>공격</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>특공</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SpecialAttack">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제우스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>포세이돈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아테나</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아프로디테</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아르테미스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아레스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디오니소스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데메테르</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>돌진</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Dash">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>590</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제우스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>포세이돈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아테나</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아프로디테</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아르테미스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아레스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디오니소스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데메테르</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>지원</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Support">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>590</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제우스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>포세이돈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아테나</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아프로디테</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아르테미스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아레스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디오니소스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데메테르</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>마법</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Magic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>포세이돈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아프로디테</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>아레스</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디오니소스</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>630</y>
      <width>77</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>마법2</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SpecialMagic">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>630</y>
      <width>97</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>[제우스]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>전기마법</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>천둥마법</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>피해강화</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>[아테나]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>탄환마법</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>[데메테르]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>마법광선</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>선택에 따라서 가능성이 있는 듀오</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>741</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="UnSelected">
     <item row="2" column="3">
      <widget class="QLabel" name="label_22">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>갈라진 사격</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_20">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>저온융합</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_18">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>악사의 저주</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_19">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>특권</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_21">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>눈보라탄환</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>갈망의 저주</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_24">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>자비로운 최후</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>비통함</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_23">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>치명적인 반격</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>계산된 위험</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>끈질긴 생명력</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_16">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>달콤한 넥타르</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_15">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>요지부동 기개</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_17">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>신기루 사격</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_33">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>수정같은 선명함</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_32">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>쪼개지는 두통</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_35">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>결빙 소용돌이</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_34">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>메스꺼움의 저주</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_13">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>복수심에 불타는 기분</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_10">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>벼락 방진</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_29">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>차가운 포옹</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>바다 폭풍</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_11">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>그을린 공기</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_12">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>피뢰침</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_31">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>사냥하는 칼날</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>반짝이는 축제</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_30">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>차가운 포옹</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_36">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>얼음 포도주</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>이미 선택되어 나타날 듀오</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>741</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="UnSelected_2">
     <item row="2" column="3">
      <widget class="QLabel" name="label_37">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>갈라진 사격</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_38">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>저온융합</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_39">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>악사의 저주</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_40">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>특권</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_41">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>눈보라탄환</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>갈망의 저주</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_43">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>자비로운 최후</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>비통함</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_45">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>치명적인 반격</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>계산된 위험</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>끈질긴 생명력</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_48">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>달콤한 넥타르</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_49">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>요지부동 기개</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_50">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>신기루 사격</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_51">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>수정같은 선명함</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_52">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>쪼개지는 두통</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_53">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>결빙 소용돌이</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_54">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>메스꺼움의 저주</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_55">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>복수심에 불타는 기분</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_56">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>벼락 방진</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_57">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>낮은 내성</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_58">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>바다 폭풍</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_59">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>그을린 공기</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_60">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>피뢰침</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_61">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>사냥하는 칼날</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_62">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>반짝이는 축제</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_63">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>차가운 포옹</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_64">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>얼음 포도주</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
